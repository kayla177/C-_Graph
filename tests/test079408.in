ENTITY EX2O5UBU name_EX2O institution
ENTITY 79LGGXFC name_79LG paper
ENTITY LRBKDFEE name_LRBK institution
ENTITY UENSPOXW name_UENS institution
ENTITY EH4X96JH name_EH4X author
ENTITY VPWQ3Q05 name_VPWQ institution
ENTITY QEP6NXIL name_QEP6 institution
ENTITY 417CN28T name_417C paper
ENTITY TRPPF51K name_TRPP conference
ENTITY MY0RA1IW name_MY0R journal
RELATIONSHIP 417CN28T authored_by TRPPF51K 3.1
RELATIONSHIP UENSPOXW collaborated_with TRPPF51K 0.9
RELATIONSHIP MY0RA1IW published_in 417CN28T 0.7
RELATIONSHIP EH4X96JH affiliated_with MY0RA1IW 8.1
RELATIONSHIP VPWQ3Q05 affiliated_with 417CN28T 8.8
RELATIONSHIP EH4X96JH authored_by QEP6NXIL 5.6
RELATIONSHIP VPWQ3Q05 collaborated_with QEP6NXIL 6.1
RELATIONSHIP MY0RA1IW authored_by QEP6NXIL 6.1
RELATIONSHIP 79LGGXFC cited_by UENSPOXW 0.3
RELATIONSHIP MY0RA1IW affiliated_with 79LGGXFC 6.9
RELATIONSHIP MY0RA1IW collaborated_with UENSPOXW 1.8
RELATIONSHIP LRBKDFEE cited_by TRPPF51K 3.6
RELATIONSHIP EX2O5UBU collaborated_with LRBKDFEE 4.5
RELATIONSHIP UENSPOXW published_in 417CN28T 5.7
RELATIONSHIP VPWQ3Q05 presented_at MY0RA1IW 9.1
PRINT VPWQ3Q05
PATH EH4X96JH UENSPOXW
DELETE MY0RA1IW
DELETE LRBKDFEE
HIGHEST
DELETE 79LGGXFC
PATH VPWQ3Q05 TRPPF51K
PRINT UENSPOXW
DELETE QEP6NXIL
HIGHEST
EXIT
