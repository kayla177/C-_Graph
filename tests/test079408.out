success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
417CN28T QEP6NXIL MY0RA1IW
EH4X96JH MY0RA1IW VPWQ3Q05 417CN28T UENSPOXW 31.7
success
success
79LGGXFC EH4X96JH 26.5
success
VPWQ3Q05 417CN28T TRPPF51K 11.9
TRPPF51K 417CN28T
success
UENSPOXW VPWQ3Q05 14.5
